dev/com.ibm.ws.security.jwt/src/com/ibm/ws/security/jwt/internal/ConsumerUtil.java

```diff
@@ -536,17 +539,18 @@ void validateClaims(JwtClaims jwtClaims, JwtContext jwtContext, JwtConsumerConfi
             }
         }
 
-        long tokenAge = 0;
+        long tokenAgeInMilliSeconds = 0;
         // Take tokenAge value from server.xml
-        tokenAge = config.getTokenAge();
+        tokenAgeInMilliSeconds = config.getTokenAge();
+
         /**
-         * If tokenAge from server.xml is zero, then take the value of tokenAge
-         * from mpConfigProps
+         * If tokenAge from server.xml is zero (disabled), then take the value of tokenAge from
+         * mpConfigProps
          */
-        if (tokenAge == 0) {
+        if (tokenAgeInMilliSeconds == 0) {
```

Comment:
Noting here since it was initially a question in my head: The `tokenAge` attribute in the `<mpJwt>` element is defined (before this PR) as:
```XML
<AD id="tokenAge" name="internal" description="internal use only" required="false" type="String" default="0s" ibm:type="duration" />
```
This PR also updates this attribute to set `min="0"`, so a negative value should fall back to the default which in this case would be 0 seconds. So as far as this code goes, setting `tokenAge="-1"` in the server.xml will behave as if `tokenAge="0s"`.